java/src/jmri/profile/ProfileProperties.java

```diff
@@ -30,15 +30,11 @@ public ProfileProperties(Profile project) {
     }
 
     @Override
-    @SuppressFBWarnings(value = "deprecation", justification = "Avoids errors passing partly constructed object.")
-    @SuppressWarnings("deprecation") // silence deprecation notice in IDEs
     public String get(String key, boolean shared) {
```

Comment:
What's your intent here? To get the methods in this class to generate deprecation warnings? I'm not against the change, I'm only trying to understand what the change is about.

Reply:
These weren't actually protecting against any warnings, at least for me.